MathToolsLibrary/Matrix.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MathToolsLibrary
+{
+    class Matrix
+    {
+        public int Rows { get; private set; }
+        public int Columns { get; private set; }
+        public int[,] Elements { get; private set; }
+
+        public Matrix(int rows, int columns)
+        {
+            Elements = new int[rows, columns];
+        }
+
+        public int GetElementAt(int row, int column)
+        {
+            return Elements[row, column];
+        }
+        
+        public void SetElementAt(int value, int row, int column)
+        {
+            Elements[row, column] = value;
+        }
+
+        public Matrix AddMatrices(Matrix matrixA, Matrix matrixB)
+        {
+            if (matrixA.Rows != matrixB.Rows || matrixA.Columns != matrixB.Columns)
+            {
+                Console.WriteLine("Matrices cannot be added");
+                return null;
+            }
+
+            Matrix result = new Matrix(matrixA.Rows, matrixA.Columns);
+
+            for (int i = 0; i < result.Rows; i++)
+            {
+                for (int j = 0; j < Columns; j++)
+                {
+                    result.Elements[i, j] = matrixA.Elements[i, j] + matrixB.Elements[i, j];
+                }
+            }
+
+            return result;
+        }
+
+        public Matrix MultiplyMatrices(Matrix matrixA, Matrix matrixB)
+        {
+            if (matrixA.Rows != matrixB.Columns)
```

Comment:
Create a generic method for handle exception... for example 
``` C#
public void HandleException(condition,message{
if(condition)
{
        throw new System.ArgumentException($"message");
}
}
```

Reply:
Done!